crates/astria-sequencer-types/src/sequencer_block_data.rs

```diff
@@ -97,12 +122,23 @@ impl SequencerBlockData {
 
     #[allow(clippy::type_complexity)]
     #[must_use]
-    pub fn into_values(self) -> (Hash, Header, Option<Commit>, HashMap<Namespace, RollupData>) {
+    pub fn into_values(
+        self,
```

Comment:
Is this formatting correct? Feels weird to push the self to new line because the return is